README.md

```diff
@@ -0,0 +1,331 @@
+<b>IMPORTANT! Make sure that you replace /dev/sdx for the right device name, e.g. /dev/sdb</b>
+
+Unmount the filesystem before creating new partition table:
+
+    $ sudo umount /dev/sdx?
+
+Create new partition table:
+
+    $ sudo fdisk /dev/sdx
+    > enter 'o' to create a new empty DOS partition table
+    > enter 'n' to create a new partition; just press return at every prompt to accept the default values
+    > enter 'w' to write the new partition table and exit
+    $ sudo mkfs.ext4 /dev/sdx1
+
+Mount the usb drive:
+
+    $ sudo mount /dev/sdx1 /mnt/
+
+To put filesystem on USB you will need 'openwrt-pistachio-marduk-marduk_cc2520-rootfs.tar.gz' which is available at "openwrt/bin/pistachio"
+
+Extract the openwrt-pistachio-marduk-marduk_cc2520-rootfs.tar.gz onto the partition you just created
+
+    # sudo rm -rf /mnt/*
+    # sudo tar -xf bin/pistachio/openwrt-pistachio-marduk-marduk_cc2520-rootfs.tar.gz -C /mnt/
+    # sudo umount /mnt/
+
+
+Run "sync" command to synchronize the data properly on the USB drive.
+
+    # sync
+
+
+Connect the USB drive and power on the board.
+
+Press Enter on the serial console to cancel the autoboot and to get the uboot prompt. Check if the all the environment variables are setup properly on the uboot prompt or not
+
+    $ pistachio # printenv
+
+Type following command to boot from usb.
+
+    $ pistachio # run usbboot
+
+## Boot from SD Card
+Similar to the instructions for Boot from usb drive, you need to format the SD card into ext4 parition. Follow the steps mentioned above by making subtle changes
+
+
+`$ mount | grep /media
+/dev/sdc1 on /media/user/5c0dac14-1d47-4a39-8b06-e27861473670 type ext4 (rw,nosuid,nodev,uhelper=udisks2)'
+
+SD media essentially mounts at /dev/sdc1 on a computer.  However, make sure to double-check if it gets somehow mounted differently on your computer, in which case, <b>IMPORTANT! Make sure that you replace /dev/sdc with /dev/sdx for the right device name</b>
+
+Unmount the filesystem before creating new partition table:
+
+    $ sudo umount /dev/sdc1
+
+Create new partition table:
+
+    $ sudo fdisk /dev/sdc
+    > enter 'o' to create a new empty DOS partition table
+    > enter 'n' to create a new partition; just press return at every prompt to accept the default values
+    > enter 'w' to write the new partition table and exit
+    $ sudo mkfs.ext4 /dev/sdc1
+
+Mount the SD media:
+
+    $ sudo mount /dev/sdc1 /mnt/
+
+To put filesystem on SD you will need 'openwrt-pistachio-marduk-marduk_cc2520-rootfs.tar.gz' which is available at "openwrt/bin/pistachio"
+
+Extract the openwrt-pistachio-marduk-marduk_cc2520-rootfs.tar.gz onto the partition you just created
+
+    # sudo rm -rf /mnt/*
+    # sudo tar -xf bin/pistachio/openwrt-pistachio-marduk-marduk_cc2520-rootfs.tar.gz -C /mnt/
+    # sudo umount /mnt/
+
+Run "sync" command to synchronize the data properly on the SD card.
+
+    # sync
```

Comment:
Replace # with $ for consistency with other commands in this file to be run on developer's PC